src/item/views/Item.tsx

```diff
@@ -165,15 +174,13 @@ const Item: FunctionComponent<ItemProps> = ({ history, match }) => {
 								<ToolbarItem>
 									<Spacer margin="bottom">
 										<div className="c-content-type c-content-type--video">
-											{itemMetaData.type && (
-												<Icon
-													name={
-														(itemMetaData.type.id === ContentTypeNumber.audio
-															? 'headphone'
-															: itemMetaData.type.label) as IconName
-													}
-												/>
-											)}
+											<Icon
+												name={
+													(itemMetaData.type.id === ContentTypeNumber.audio
```

Comment:
heeft een itemMetaData object altijd een type en een id? of gebruik je hier ook best _.get?